### PR TITLE
adjust results logic for MSRPOPLEX and add tests

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR fills out the logic for results so that when MSRPOPL is 0, MSRPOPLEX is pulled to 0.  Tests were added to ensure that this is true as well as MSRPOPLEX being unmodified if MSRPOPL is 1.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1219
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] N/A If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered. 
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] N/A JIRA test links:
- [x] N/A Justification for using JIRA tests:
- [x] N/A JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: app/assets/javascripts/cql_calculator.js.coffee [hunk 1]
```diff
@@ -196,6 +196,9 @@
         population_results['NUMER'] = 0
       if 'NUMEX' of population_results
         population_results['NUMEX'] = 0
+      # if MSRPOPL = 1, then MSRPOPLEX can be 0 or 1
+      if 'MSRPOPLEX' of population_results && @isValueZero('MSRPOPL', population_results)
```
Comment (hossenlopp) [inline app/assets/javascripts/cql_calculator.js.coffee]: The  `&& @isValueZero('MSRPOPL', population_results)` isn't needed because it is checked above.

**Review by hossenlopp — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/cql_calculator.js.coffee (hunk 1 above)
Comment (lukoktonos) [reply app/assets/javascripts/cql_calculator.js.coffee]: The `&& @isValueZero('MSRPOPL', population_results)` is there to cover the case when `DENOM == 0` but `MSRPOPL == 1`.  We only want to zero out `MSRPOPLEX` if `MSRPOPL == 0`.

Comment (hossenlopp) [reply app/assets/javascripts/cql_calculator.js.coffee]: DENOM and MSRPOPL shouldn't exist on a measure at the same time.

**Review by jbradl11 — APPROVED** [review]
Simple change. Tests pass. Code looks good.

**Review by hossenlopp — APPROVED** [review]

**Pull request merged**